Assets/General/Scripts/Health/GeneralVfxFunctionality.cs

```diff
@@ -46,7 +50,17 @@ public void StartAllVfx()
             animator.SetTrigger(VFX_ANIMATION_TRIGGER);
         }
     }
-
+/// <summary>
+/// To move Vfx forward if need be
+/// </summary>
+    void Update()
```

Comment:
Personally would prefer this as a coroutine, but I can't force ya

Reply:
I tried coroutine at first but it kept exploding so I gave up on it